pulsar-broker-common/pom.xml

```diff
@@ -51,5 +51,11 @@
       <groupId>javax.servlet</groupId>
       <artifactId>javax.servlet-api</artifactId>
     </dependency>
+
+    <dependency>
```

Comment:
I' ll move athenz plugin codes to another module directory(like "pulsar-auth-athenz"), thanks.